dm/loader/util.go

```diff
@@ -45,14 +46,26 @@ func percent(a int64, b int64, finish bool) string {
 }
 
 func getMydumpMetadata(ctx context.Context, cli *clientv3.Client, cfg *config.SubTaskConfig, workerName string) (string, string, error) {
```

Comment:
The source config should have set flavor https://github.com/pingcap/tiflow/blob/d105a2b4b780a1d4b6dd9bbf6f43bd7af081efdf/dm/config/source_config.go#L335-L346 . Maybe it's propagated as https://github.com/pingcap/tiflow/blob/d105a2b4b780a1d4b6dd9bbf6f43bd7af081efdf/dm/config/subtask.go#L118